Screengrabfile

```diff
@@ -0,0 +1,21 @@
+# remove the leading '#' to uncomment lines
+
+app_package_name 'org.mozilla.focus.debug'
+use_tests_in_packages ['org.mozilla.focus.activity']
+
+app_apk_path 'app/build/outputs/apk/app-focus-webkit-debug.apk'
+tests_apk_path 'app/build/outputs/apk/app-focus-webkit-debug-androidTest.apk'
+
+# 63 locales
+locales ['ar', 'ast', 'az', 'bg', 'bn-BD', 'bn-IN', 'ca', 'cs', 
+'cy', 'da', 'de', 'dsb', 'el', 'eo', 'es-AR', 'es-CL', 'es-ES', 'es-MX', 'eu',
+'fa', 'fr', 'fy-NL', 'ga-IE', 'he', 'hi-IN',  'hsb', 'hu', 'hy-AM', 'id', 'it',
+'ja', 'ka', 'kab', 'kk', 'ko', 'lo', 'mix', 'ms', 'my','nb-NO', 'ne-NP', 'nl',
+'nn-NO', 'pl', 'pt-BR', 'ro', 'ru', 'sk', 'sl', 'sq', 'sr', 'sv-SE', 'ta', 'te', 
+'th', 'tr','uk', 'ur','zh-CN', 'zh-TW','en-US']
+
```

Comment:
So I assume there would be a manual process to check whether the locale lists are up to date? We should clarify the steps in a wiki somewhere.

Reply:
Yeah, we have multiple lists of locales that I want to make more explicit somewhere:
* A: List of locales we ship in our dev/beta builds
* B: List of locales we ship in releases (B < A)
* C: List of locales we take screenshots of (C = A)